### ISSUE TITLE
[COGNITION] CONSIDER adding a heading before "Longer than 1 year"

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#7847]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On Fiduciary claims — [https://preview-prod.vfs.va.gov/preview?nodeId=3016](https://preview-prod.vfs.va.gov/preview?nodeId=3016),  **consider** adding a heading before "Longer than 1 year", as without the context it may be confusing for screen reader users who don't have the context of the visual design.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to understand the context of the content as sighted users gain from visual design.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destination: devshop?/prod?

## Steps to Recreate

1. Enter https://preview-prod.vfs.va.gov/preview?nodeId=3016 in browser, in the VA environment
2. Start screenreading device listed in Environment
3. Navigate to the section "Appeal to a Veterans Law Judge by requesting a Board Appeal"
4. Notice the first "About 1 year" has a heading "How long it take VA to make a decision?"
5. Verify the second "About 1 year" doesn't have the heading, and sounds abrupt and confusing.

## Possible Fixes (optional)

Add  "How long it take VA to make a decision?" before the second instance of "About 1 year".

## WCAG or Vendor Guidance (optional)

* [Web Accessibility In Context](https://www.smashingmagazine.com/2019/06/web-accessibility-context/)
* [Designing For Accessibility And Inclusion](https://www.smashingmagazine.com/2018/04/designing-accessibility-inclusion/)

## Screenshots or Trace Logs

![image](https://user-images.githubusercontent.com/57469/78994376-e2516d80-7b0d-11ea-9bd8-1437b8f50bc6.png)

**Comment by lwalsh617** [comment]
Page content was edited and reformatted. Number component is no longer being used so issue has been resolved. Screenshot attached. Closing ticket. 
![Screen Shot 2020-05-20 at 8 07 48 AM](https://user-images.githubusercontent.com/53535663/82444421-39355580-9a71-11ea-8acf-9d8227606f9f.png)

**Comment by jenstrickland** [comment]
Validated. Thanks for addressing this!
